js/bytetrade.js

```diff
@@ -0,0 +1,1000 @@
+            const quoteNameAndId = quoteName + '@' + quoteId;
+            let base = '';
+            let quote = '';
+            if (baseNameAndId in this.commonCurrencies) {
+                base = this.commonCurrencies[baseNameAndId];
+            } else {
+                base = baseName;
+            }
+            if (quoteNameAndId in this.commonCurrencies) {
+                quote = this.commonCurrencies[quoteNameAndId];
+            } else {
+                quote = quoteName;
+            }
+            const symbol = base + '/' + quote;
+            const amountMin = this.safeFloat (market['limits']['amount'], 'min');
+            const amountMax = this.safeFloat (market['limits']['amount'], 'max');
+            const priceMin = this.safeFloat (market['limits']['price'], 'min');
+            const priceMax = this.safeFloat (market['limits']['price'], 'max');
+            const precision = {
+                'amount': market['precision']['amount'],
+                'price': market['precision']['price'],
+            };
+            const status = this.safeString (market, 'active');
+            const active = (status === true);
+            const entry = {
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'info': market,
+                'active': active,
+                'precision': precision,
+                'limits': {
+                    'amount': {
+                        'min': amountMin,
+                        'max': amountMax,
+                    },
+                    'price': {
+                        'min': priceMin,
+                        'max': priceMax,
+                    },
+                    'cost': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                },
+            };
+            result.push (entry);
+        }
+        return result;
+    }
+
+    async fetchBalance (params = {}) {
+        if (!('userid' in params) && (this.apiKey === undefined)) {
+            throw new ArgumentsRequired ('fetchDeposits requires hasAlreadyAuthenticatedSuccessfully or userid argument');
+        }
+        await this.loadMarkets ();
+        const request = {};
+        if ('userid' in params) {
+            request['userid'] = params['userid'];
+        } else {
+            request['userid'] = this.apiKey;
+        }
+        const balances = await this.publicGetBalance (request);
+        const result = { 'info': balances };
+        // const balances = this.safeValue (response, 'list', []);
+        for (let i = 0; i < balances.length; i++) {
+            const balance = balances[i];
+            const currencyId = this.safeString (balance, 'code');
+            let code = currencyId;
+            if (currencyId in this.currencies_by_id) {
```

Comment:
use `const code = this.safeCurrencyCode (currencyId)` instead

Reply:
thank you, i have fixed it.